README.md

```diff
@@ -13,10 +16,18 @@ Run the build:
 
 `mvn install`
 
-The build runs copyright check and generates the jar, sources-jar and javadoc-jar by default.
+The build runs a copyright check and generates the jar, sources-jar and javadoc-jar by default.
+
+The API jar can be found in /api/target.
 
 Checking findbugs
 -----------------
 
 `mvn -DskipTests -Dfindbugs.threshold=Low findbugs:findbugs`
 
+About Jakarta Servlet
+-------------
+
+Jakarta Servlet defines a server-side API for handling HTTP requests and responses,
+and a packaging model for such server-side applications.
```

Comment:
The decision from #239  was that this should just be:

> Jakarta Servlet defines server-side handling for HTTP requests and responses.

Reply:
I think this is just the description, not the official scope. It can go beyond what the scope says, as it's just informative to the one reading what the project is about.

Reply:
I think we should just stick with the official scope, at least until we've had a discussion about what we want to say here.